source/edge-server/connect.txt

```diff
@@ -83,10 +116,10 @@ modifications you can make to the :github:`Swift template app
            - Value
 
          * - ``baseUrl``
-           - The public-accessible DNS of your Edge Server's host.
+           - The public-accessible DNS of your Edge Server instance host.
              This is ``http://``, or if TLS is enabled, ``https://``, 
-             followed by the same value you set in your Edge Server 
-             :file:`edge_config.json` ``baseURL`` field.
+             followed by the same value you set in your Edge Server
+             instance configuration.
```

Comment:
this is not somehting they'd set in their configuration, rather its the ip address exposed by their docker or kubernetes cluster (so maybe can describe this as their "container configuration" instead?